packages/assets-controllers/src/TokenBalancesController.test.ts

```diff
@@ -3,6 +3,7 @@ import { toHex } from '@metamask/controller-utils';
 import BN from 'bn.js';
 
 import { flushPromises } from '../../../tests/helpers';
+import { createMockInternalAccount } from '../../accounts-controller/src/tests/mocks';
```

Comment:
Not too sure whats the best approach for this. If we move the mocks to the shared `tests` directory it would require us to add `@metamask/keyring-api` and `@metamask/keyring-controller` as a dependency and create a circular dependency issue. 

One solution may be to duplicate the code thats needed for the mock to work.

Reply:
Hmm good point, lets not block this PR on this. We can figure it out in a future PR